src/Microsoft.Dnx.Tooling/Publish/DependencyContext.cs

```diff
@@ -28,8 +28,8 @@ public DependencyContext(Runtime.Project project, FrameworkName targetFramework)
             PackagesDirectory = applicationHostContext.PackagesDirectory;
         }
```

Comment:
Yeah, this was left over from some other changes but is a benign refactoring that is useful for other things. I can revert it if you want.